strawberryfields/ops.py

```diff
@@ -1603,7 +1598,7 @@ class Interferometer(Decomposition):
     Args:
         U (array): an :math:`N\times N` complex unitary matrix.
         tol (float): the tolerance used when checking if the matrix is unitary:
-            :math:`|UU^\dagger-I| \leq tol`
+            :math:`|UU^\dagger-I| \leq` tol
```

Comment:
Should this be
```suggestion
            :math:`|UU^\dagger-I| \leq \text{tol}`
```

Reply:
I tried that first, but this way `tol` is rendered in the same font as above.